project_helper.rb

```diff
@@ -21,28 +21,105 @@ def initialize(project_or_workspace_path, scheme_name)
 
     # read scheme application targets
     @main_target, @targets_container_project_path = read_scheme_archivable_target_and_container_project(scheme, scheme_container_project_path)
+
+    puts "Scheme target: #{@main_target}, container path: #{@targets_container_project_path}"
+    puts 
```

Comment:
Do we need this empty puts statement here?

Reply:
Optional: Makes it easier to read the console output..